packages/block-editor/src/components/inserter/block-types-tab.native.js

```diff
@@ -21,7 +23,14 @@ function BlockTypesTab( { onSelect, rootClientId, listProps } ) {
 
 			const allItems = getInserterItems( rootClientId );
 			const blockItems = allItems.filter(
-				( { category } ) => ! NON_BLOCK_CATEGORIES.includes( category )
+				( { id, category } ) =>
+					! NON_BLOCK_CATEGORIES.includes( category ) &&
+					// We don't want to show all possible embed variations
+					// as different blocks in the inserter. We'll only show a
+					// few popular ones.
+					( category !== 'embed' ||
+						( category === 'embed' &&
+							ALLOWED_EMBED_VARIATIONS.includes( id ) ) )
```

Comment:
Normally the embed variations were hidden from the inserter as we had an empty `embed/variations.native.js` file (which I deleted in this PR). We needed to re-enable those as the information if an embed supports this setting is kept in the `attributes` there with the `responsive` flag: 
https://github.com/WordPress/gutenberg/blob/3e77c785d114e3a4cad7d34b5df66a0fbea6f1f9/packages/block-library/src/embed/variations.js#L44

This should also help us when we add a few more variations for https://github.com/wordpress-mobile/gutenberg-mobile/issues/3278